CMakeLists.txt

```diff
@@ -66,9 +66,9 @@ set(RDK_BOOST_VERSION "1.58.0")
 
 if(NOT MSVC)
   if(RDK_OPTIMIZE_POPCNT)
-    if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
+    if(CMAKE_HOST_PROCESSOR STREQUAL "aarch64")
```

Comment:
Shouldn't this also be CMAKE_SYSTEM_PROCESSOR?

Reply:
Yes, that was wrong :facepalm:  Thanks